api/RTCPeerConnection.json

```diff
@@ -565,10 +545,7 @@
               "chrome_android": {
                 "version_added": "57"
               },
-              "edge": {
-                "version_added": "≤79",
-                "notes": "Default for `rtcpMuxPolicy` is `require`"
```

Comment:
Cases like this with extra notes suggest this was available in Edge 18 or earlier.

Reply:
This note was copied from Chrome, though!

Reply:
Oops, missed that!